translate/storage/test_mo.py

```diff
@@ -124,6 +128,14 @@ def test_language(self):
         store.updateheader(add=True, Language="zu")
         assert store.gettargetlanguage() == "zu"
 
+    def test_context(self):
+        store = self.StoreClass()
+        unit = mo.mounit('source')
```

Comment:
Nit: Use `store.UnitClass` instead of `mo.mounit`.

Reply:
`AttributeError: 'TestMOFile' object has no attribute 'UnitClass'`

Should I add it or keep using mo.mounit?

Reply:
In the end I've chosen to use `self.StoreClass.UnitClass`...